### DISCUSSION TITLE
Move classic Notelock to options as togglable feature

### DISCUSSION THREAD

**Discussion by ZiolkowskiFilip** [ppy/osu#26896] [Ideas]
Hello I come with a proposition, a quality of life change of sorts. Would it be possible to make notelock a toggleable feature in options instead of being tied to classic mod?

My argument is that it would make lazer feel more like stable while still being flexible for players who don't like this mechanic while simultaneously not punishing people like myself with a 0.96x classic mod multiplayer for wanting the game to feel like stable.

In my opinion the notelock mechanic was with us for so long that some of us (me) depend on it without realizing it. My feedback is based on my Spotlight plays that were painfully hard for me given the lack of notelock.
I know it isn't a common stance to defend notelock but I would love to see such change implemented.

Taken directly from my post on the forum, as I figured that most people don't read them anymore ;-;

**Comment by Rekunan** [comment]
The devs want to avoid as many toggles as they can, due to the extra work required in mainlining them, and since you can already achieve this with classic mod, I don't think this will be implemented.

To touch on the score multiplier issue though, do keep in mind that they can be changed easily, and possibly be dynamic in the future, depending on the mod option.
